eth/handler_test.go

```diff
@@ -150,7 +149,6 @@ func newTestHandlerWithBlocks(blocks int) *testHandler {
 		Chain:      chain,
 		TxPool:     txpool,
 		Network:    1,
-		Sync:       downloader.FastSync,
```

Comment:
Probably needs to be set to `FullSync`, no?

Reply:
Yes, will change